.github/workflows/update.yml

```diff
@@ -49,6 +52,38 @@ jobs:
           git config user.name 'github-actions[bot]'
           git config user.email '41898282+github-actions[bot]@users.noreply.github.com'
 
+      - name: Get info on the current PR
+        id: open_pr_info
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          # Query for info about the already open update PR
+          info=$(
+            gh api graphql -F owner='{owner}' -F repo='{repo}' -F branch="$branch" -f query='
```

Comment:
Didn't even know github had a graphql endpoint. nice.

Reply:
Yeah, you can play with it in the online repl too: https://docs.github.com/en/graphql/overview/explorer